keyboards/handwired/oktiboard/config.h

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 Oktay Elipek
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5454  // "TT" - Teletubbies
```

Comment:
```suggestion
#define VENDOR_ID       0x5455  // "TT" - Teletubbies
```
0x5454 is already used by a VIA compatible board:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/satt/vision/config.h#L21